src/main/resources/swagger.api/schema/assignmentCheckDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Container for raw marc and target type",
+  "type": "object",
+  "properties": {
+    "rawMarc": {
+      "type": "string",
+      "description": "Raw MARC json"
+    },
+    "target": {
+      "type": "string",
+      "enum": ["CREATOR_OF_WORK"],
```

Comment:
What about SUBJECT_OF_WORK?
We have it in java enum, but not here

Reply:
fixed.